PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Install Remote Assist.md

```diff
@@ -11,52 +11,53 @@ mac
 ```
 set -euo pipefail
 
-declare -r REMOTE_DMG_URL="https://cdn.jumpcloud.com/TheJumpCloud/jumpcloud-remote-assist-agent/latest/jumpcloud-assist-app.dmg"
-declare -r LOCAL_DMG_TMP_PATH="/tmp/jumpcloud-assist-app.dmg"
-declare -r APP_NAME="Jumpcloud Assist App"
-declare -r APP_FILENAME="$APP_NAME.app"
```

Comment:
Some comments:
1. Enclose every variable reference in `${}` in order to avoid too much turn around when backporting the script to jumpcloud-devices. For example `${APP_NAME}`. Scott asked for this change and might ask for it again in jumpcloud-devices.
2. Please include the shebang `#!/usr/bin/env bash` at the beginning of the script to avoid too much turn around when backporting the script to jumpcloud-devices. Scott asked for this change and might ask for it again in jumpcloud-devices.

Reply:
Why is it assumed that `env` is always in `/usr/bin`? That isn't portable.